design-tokens/tokens/layer1/base.json

```diff
@@ -274,49 +274,77 @@
     },
     "typography": {
       "paragraph": {
-        "default": {
-          "value": {
-            "fontFamily": "{echoes.fontFamily.sans}",
-            "fontWeight": "{echoes.fontWeight.regular}",
-            "lineHeight": "{echoes.lineHeight.body}",
-            "fontSize": "{echoes.fontSize.body.medium}",
-            "letterSpacing": "{echoes.letterSpacing.default}",
-            "paragraphSpacing": "{echoes.paragraphSpacing.medium}"
+        "medium": {
+          "regular": {
```

Comment:
We need to update the code since this is referenced in some places already. I'll add a commit.